internal/component/otelcol/receiver/influxdb/influxdb.go

```diff
@@ -0,0 +1,91 @@
+// Package influxdb provides an otelcol.receiver.influxdb component.
+package influxdb
+
+import (
+	"fmt"
+
+	"github.com/grafana/alloy/internal/component"
+	"github.com/grafana/alloy/internal/component/otelcol"
+	otelcolCfg "github.com/grafana/alloy/internal/component/otelcol/config"
+	"github.com/grafana/alloy/internal/component/otelcol/receiver"
+	"github.com/grafana/alloy/internal/featuregate"
+	influxdbreceiver "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/influxdbreceiver"
+	otelcomponent "go.opentelemetry.io/collector/component"
+	otelextension "go.opentelemetry.io/collector/extension"
+	"go.opentelemetry.io/collector/pipeline"
+
+)
+
+func init() {
+	component.Register(component.Registration{
+		Name:      "otelcol.receiver.influxdb",
+		Stability: featuregate.StabilityGenerallyAvailable,
+		Args:      Arguments{},
+
+		Build: func(opts component.Options, args component.Arguments) (component.Component, error) {
+			fact := influxdbreceiver.NewFactory()
+			return receiver.New(opts, fact, args.(Arguments))
+		},
+	})
+}
+
+// Arguments configures the otelcol.receiver.influxdb component.
+type Arguments struct {
+	Endpoint string `alloy:"endpoint,attr"` // Explicitly define endpoint as a top-level field
```

Comment:
this should be removed, you already have the endpoint field coming from the HTTPServer block and because it is squashed it will be top level